### PR TITLE
Django Setup

### PR DESCRIPTION
**Pull request by rmangi**
Here's some django for ya.

make sure your requirements are up to date with the latest

```
cd Capitol-Words/capitolwords_ng/capitolweb
python manage.py runserver
```

Then you can hit these urls

http://127.0.0.1:8000/cwapi/search/multi/?speaker=ryan
http://127.0.0.1:8000/cwapi/search/multi/?speaker=schumer&title=trump
http://127.0.0.1:8000/cwapi/search/title/trump/
http://127.0.0.1:8000/cwapi/search/speaker/Schumer/

Just a few examples to start working from. This is using the pretty sweet elasticsearch_dsl package here:

https://elasticsearch-dsl.readthedocs.io/en/latest/index.html

@will-horning @jeiranj @heavi5ide

### CODE REVIEW

**Comment by heavi5ide** [comment]
Nice, thanks Rick! TBH though, not sure I actually like elasticsearch-dsl -- seems unnecessary given the simplicity of elasticsearch's JSON queries, and the fact that elasticsearch documents have ad hoc structure means the DSL doesn't even do what something like Django ORM does (translating model into table/column). What exactly do we gain from it?

File: capitolwords_ng/capitolweb/cwapi/views.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.shortcuts import render
+
+from django.http import HttpResponse
+from django.conf import settings
+from django.http import JsonResponse
+from elasticsearch_dsl import Search
+from elasticsearch_dsl.query import Match, Q
```
Comment (will-horning) [inline capitolwords_ng/capitolweb/cwapi/views.py:7]: nit: separate lines

**Comment by will-horning** [comment]
LGTM, after this we just need support for searching on topic and full text. We can do topic now with whats already in there, I'll work on getting full text data in there today.

**Comment by rmangi** [comment]
@heavi5ide on the elasticsearch-dsl side it's much easier to compose more complex queries in python and it allows us to search multiple indexes

File: capitolwords_ng/capitolweb/cwapi/views.py (hunk 1 above)
Comment (rmangi) [reply capitolwords_ng/capitolweb/cwapi/views.py:7]: I propose we allow multi imports on a line with python3 :)

**Comment by rmangi** [comment]
Lots of new stuff. 

I set up models and loaded data for all of the congress people, their terms (when they served), party affiliation, etc. All the basic stuff from unitedstates/congress-legislators

I created a json endpoint for fetching a congress person by name as well.

To use this:

in Capitol-Words/capitolwords_ng/capitolweb
```
python manage.py migrate
python manage.py loadcongress
python manage.py createsuperuser (if you want to use the admin tool to browse)
python manage.py runserver

Then you can browse to:
http://127.0.0.1:8000/admin
http://127.0.0.1:8000/legislators/person/kennedy/  (returns John and Joe)
```

**Comment by heavi5ide** [comment]
@rmangi I had to `pip install pyyaml` to get `loadcongress` to work -- might need to be added to requirements.txt?

**Comment by heavi5ide** [comment]
We should also start a `capitolwords_ng/.gitignore` with at least `capitolweb/db.sqlite3` in there

**Comment by rmangi** [comment]
merged

**Pull request closed without merging**